libvuln/init.go

```diff
@@ -28,6 +29,12 @@ func initUpdaters(ctx context.Context, opts *Opts, db *sqlx.DB, store vulnstore.
 			eC <- fmt.Errorf("duplicate updater found in UpdaterFactory. all names must be unique: %s", u.Name())
 			return
 		}
+		if c, ok := u.(driver.Configurable); ok {
```

Comment:
I think this is going to take a little more thinking about. This is going to be an issue where we generate the names dynamically like Oracle (generating the date).